how-to/maintain/backup-and-restore/mydumper-lightning.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Use `Mydumper` and `TiDB Lightning` for Backup and Restoration
+category: how-to
+aliases: ['/docs-cn/how-to/maintain/backup-and-restore/']
+---
+
+# Use `Mydumper` and `TiDB Lightning` for Data Backup and Restoration
+
+This document describes how to take full back up and restore the data of TiDB using Mydumper and TiDB Lightning. For incremental backup and restoration, refer to [TiDB Binlog](/reference/tidb-binlog/overview.md).
+
+Suppose that the TiDB service information is as follows:
+
+|Name|Address|Port|User|Password|
+|----|-------|----|----|--------|
+|TiDB|127.0.0.1|4000|root|*|
+
+Use the following tools for data backup and restoration:
+
+- [Mydumper](/reference/tools/mydumper.md): to export data from TiDB
+- [TiDB Lightning](/reference/tools/tidb-lightning/overview.md): to import data into TiDB
+
+## Full backup and restoration using `Mydumper`/`TiDB Lightning`
+
+`mydumper` is a powerful data backup tool. For more information, see [`maxbube/mydumper`](https://github.com/maxbube/mydumper).
+
+Use [Mydumper](/reference/tools/mydumper.md) to export data from TiDB and use [TiDB Lightning](/reference/tools/tidb-lightning/overview.md) to import data into TiDB.
+
+> **Note:**
+>
+> It is recommended to download [Mydumper](/reference/tools/mydumper.md) from the PingCAP website, since the R&D team adapted `mydumper` for TiDB. It is not recommended to use `mysqldump` which is much slower for both backup and restoration.
```

Comment:
In technical documents, it is recommended to use `because` rather than `since` or `as` to convey the casual relationship. `since` or `as` has multiple meanings.
```suggestion
> It is recommended to download [Mydumper](/reference/tools/mydumper.md) from the PingCAP website, because the R&D team has adapted `mydumper` for TiDB. It is not recommended to use `mysqldump` which is much slower for both backup and restoration.
```